src/core/ledger/ledger.js

```diff
@@ -575,6 +602,52 @@ export class Ledger {
     account.identity = updatedIdentity;
   }
 
+  /**
+   * setPayoutAddress allows participants to set a payable address.
+   * These addresses are keyed on a specific currency, which ensures that
+   * users don't erroneously receive a grain distribution to an address that
+   * cannot handle it (such as a custodial wallet, or rigidly-designed
+   * contract) and effectively lose that reward.
+   *
+   * An address may be deleted by passing in an empty string for the
+   * `address` parameter. This is useful in case the underlying private key
+   * is compromised or the exchange hosting a custodial account is hacked.
+   */
+  setPayoutAddress(
+    id: IdentityId,
+    currency: CurrencyAddress,
+    address: EthAddress | ""
```

Comment:
why pass in `""` as the delete param instead of `null`?

Reply:
type-consistency but null makes sense